deploy/olm-catalog/keycloak-operator/7.0.1/keycloakbackups.keycloak.org.crd.yaml

```diff
@@ -0,0 +1,77 @@
+kind: CustomResourceDefinition
+metadata:
+  name: keycloakbackups.keycloak.org
+spec:
+  group: keycloak.org
+  names:
+    kind: KeycloakBackup
+    listKind: KeycloakBackupList
+    plural: keycloakbackups
+    singular: keycloakbackup
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: KeycloakBackup is the Schema for the keycloakbackups API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+          required: []
+          properties:
+            aws:
+              description: 'A set of settings for performing backups (or periodic backups) to AWS S3 bucket'
+              type: object
+              properties:
+                schedule:
+                  description: 'An optional Cron expression defining at what time the database snapshots will be taken. Here''s an example "*/1 * * * *"'
+                  type: string
+                credentialsSecretName:
+                  description: |
+                    A name of a Secret holding AWS credentials. The Secret should be in the following format:
+
+                    apiVersion: v1
+                    kind: Secret
+                    metadata:
+                      name: <Secret name>
+                    type: Opaque
+                    stringData:
+                      AWS_S3_BUCKET_NAME: <Your S3 Bucket Name>
+                      AWS_ACCESS_KEY_ID: <Your Access Key ID>
+                      AWS_SECRET_ACCESS_KEY: <Your Secret Access Key>
+                  type: string
+                encryptionKeySecretName:
+                  description: |
+                    A name of a Secret holding encryption credentials. The Secret should be in the following format:
+
+                    apiVersion: v1
+                    kind: Secret
+                    metadata:
+                      name: <Secret name>
+                    type: Opaque
+                    stringData:
+                      GPG_PUBLIC_KEY: <Your GPG Public Key>
+                      GPG_TRUST_MODEL: <Your GPG Trust Model>
+                      GPG_RECIPIENT: <Your GPG Recipient>
+                  type: string
+        status:
+          type: object
+      type: object
```

Comment:
this should be removed to allow it to work on K8s 1.11. Not sure if that is needed since this is just olm though.